akka-diagnostics/src/test/scala/akka/diagnostics/ConfigCheckerSpec.scala

```diff
@@ -74,6 +74,18 @@ class ConfigCheckerSpec extends AkkaSpec {
 
   "The ConfigChecker" must {
 
+    "find no warnings in PersistenceTestKitPlugin.scala configuration" in {
+      val extendedReference = ConfigFactory.load("persistence.conf").withFallback(reference)
+      val c = ConfigFactory
+        .parseString("""
+        akka.persistence.testkit.snapshotstore.pluginid.snapshot-is-optional = false
+        """)
+        .withFallback(reference)
+
+      val checker = new ConfigChecker(extSys, c, extendedReference)
```

Comment:
persistence.conf doesn't exist, and this should just use `reference` instead of `extendedReference`

Reply:
My bad. Didn't push it. But I see the comment below